pkg/keys/ecdsa_test.go

```diff
@@ -41,6 +77,32 @@ func (ECDSASuite) TestUnmarshalECDSA(c *C) {
 	c.Assert(verifier.UnmarshalPublicKey(badKey), IsNil)
 }
 
+func FuzzInvalidUnmarshalECDSA(f *testing.F) {
+	f.Fuzz(func(t *testing.T, s string) {
+		if len(s) > keySize {
+			// len(s) > 40 is because 256 / 8 = 32, and 32 + 8 = 40, this is from the bitsize of the curve, we skip
+			// this because only then will we get bad data
```

Comment:
The goal of that was to avoid `keySize` larger than 40 so that the fuzzer doesn't pass that data. I still don't get your comment. Could you please explain?

Reply:
Oops, I read the `>` in the check backwards! This is fine, feel free to resolve.